packages/fxa-auth-server/lib/senders/emails/templates/verifyLoginCode/en.ftl

```diff
@@ -1,6 +1,9 @@
 # Variables:
 #  $serviceName (String) - A service the user hasn't signed into before (e.g. Firefox)
-verifyLoginCode-subject-line = Sign-in code for { $serviceName }
-verifyLoginCode-title = Is this you signing in?
-verifyLoginCode-prompt-2 = If yes, here is the confirmation code:
+verifyLoginCode-subject-line-2 = Approve sign in to { $serviceName }
+verifyLoginCode-title-2 = Did you sign in to { $serviceName }?
+# Information such as browser, IP address, date and time of the request that
+# triggered the email follows.
+verifyLoginCode-request-info = Help us keep your account safe by approving you signed in on:
+verifyLoginCode-prompt-3 = If yes, here is your authorization code:
```

Comment:
```suggestion
verifyLoginCode-prompt-3 = Here is your authorization code:
```
It appears that `verifyLoginCode-title = Is this you signing in?` is being deleted, so `"If yes,` doesn't seem to make sense in this string anymore.